src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,18 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
+        if (result1 == bet1 && result2 == bet2) {
```

Comment:
Please use the ternary operator in your solution.

Reply:
okay, but I guess it's better to use if statements